RtEthSample/RtEthSample.vcxproj

```diff
@@ -119,6 +118,15 @@
       <EnvironmentVariable>true</EnvironmentVariable>
     </BuildMacro>
   </ItemGroup>
+  <ItemGroup>
+    <None Include="packages.config" />
+  </ItemGroup>
   <!-- /Necessary to pick up proper files from local directory when in the IDE-->
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
+  <Target Name="EnsureNuGetPackageBuildImports" BeforeTargets="PrepareForBuild">

```

Comment:
NuGet is used to get the Windows Implementation Library which include the common.h file that is now necessary for this project. It is also the reason for the import on line 45 and for the packages.config file that was added. Not sure if all these changes are necessary or if common.h could have just been included instead.